common/locales/en/defaultTasks.json

```diff
@@ -1,21 +1,19 @@
 {
-  "defaultHabit1Text": "Productive Work (Click the pencil to edit)",
```

Comment:
No, I changed that intentionally. I can change it back. Why is this one json file different from all the other languages? Why would it matter, it's JSON?